images/linux/scripts/installers/terraform.sh

```diff
@@ -5,7 +5,7 @@
 ################################################################################
 
 # Install Terraform
-TERRAFORM_VERSION=$(curl -s https://checkpoint-api.hashicorp.com/v1/check/terraform | jq -r .current_version)
+TERRAFORM_VERSION=$(curl -s https://api.releases.hashicorp.com/v1/releases/terraform/latest | jq -r .version)
 curl -LO "https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip"
```

Comment:
Replace curl to the helper function `download_with_retries` [source $HELPER_SCRIPTS/install.sh]

Reply:
updated